scripts/package

```diff
@@ -28,6 +28,10 @@ TAG="v${APIVERSION}_`date -u +%Y%m%d`"
 REPO=${REPO:-longhornio}
 IMAGE=${REPO}/${PROJECT}:${TAG}
 
+# update base image to get latest changes
+BASE_IMAGE=`grep FROM package/Dockerfile | grep -vi AS | awk '{print $2}'`
```

Comment:
Does this only refer to the first image, now that run doesn't have the AS clause anymore?

Reply:
Yes.

Reply:
This line will ensure image `` will be updated before building, and it will utilize the caching layers in case of no update has been introduced to the image.

```
30: Pulling from fedora-minimal
--
6011 | Digest: sha256:5093ce3d2a3b37888f85579a4f89ad5c17bdfd1eab9489203bb7af2ac1455394
6012 | Status: Image is up to date for registry.fedoraproject.org/fedora-minimal:30
6013 | Sending build context to Docker daemon   56.2MB
```